internal/action/show.go

```diff
@@ -51,6 +51,8 @@ func showParseArgs(c *cli.Context) context.Context {
 
 	if c.IsSet("alsoclip") {
 		ctx = WithAlsoClip(ctx, c.Bool("alsoclip"))
+	} else {
```

Comment:
We could save the else clause if we move line 55 before line 52 (outside of the if block). That way the default would always come from `show.autoclip` but the `--alsoclip` flag can still override it.